portfolio-project-matching-app/components/UserIcon.js

```diff
@@ -0,0 +1,18 @@
+import Image from 'next/image'
+
+const UserIcon = ({ username, imgPath }) => {
```

Comment:
Not sure we will end up using this, but I thought it looked kinda cool in Josh's browseProjects wireframe. Would look kinda neat if the little user tiles had an icon to the left w/ user name to the right. Could add the ability to upload a custom icon to Firebase Firestore as a stretch goal (similar to what we did with Wastegram when you upload the picture and just work with the URL going forward).